fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -13,6 +13,7 @@ import { Timestamp } from 'model/objects';
 import TextBlock from 'components/TextBlock';
 import DropdownSelector from 'components/DropdownSelector';
 import TextInputList from 'components/TextInputList';
+import { requestCreateElection } from '../../../../../network';
```

Comment:
```suggestion
import { requestCreateElection } from 'network';
```
would be more readable :)